internal/controller/nonadminbackup_controller_test.go

```diff
@@ -149,107 +148,108 @@ func deleteTestNamespaces(ctx context.Context, nonAdminNamespaceName string, oad
 
 var _ = ginkgo.Describe("Test single reconciles of NonAdminBackup Reconcile function", func() {
 	var (
-		ctx                   = context.Background()
-		nonAdminNamespaceName = ""
-		oadpNamespaceName     = ""
-		counter               = 0
-		updateTestScenario    = func() {
-			counter++
-			nonAdminNamespaceName = fmt.Sprintf("test-nonadminbackup-reconcile-%v", counter)
-			oadpNamespaceName = nonAdminNamespaceName + "-oadp"
-		}
+		ctx                     = context.Background()
+		nonAdminObjectName      string
+		nonAdminObjectNamespace string
+		oadpNamespace           string
+		veleroBackupNameUUID    string
```

Comment:
would remove this variable, we only need it in `scenario.uuidFromTestCase == true`, not for all cases

Reply:
I prefer to keep all the input or state vars to keep in one place. It's easier to maintain then having test getting some vars from one place and then injecting another from another place.